onnxruntime/contrib_ops/cpu/quantization/bestla_gemm.h

```diff
@@ -0,0 +1,58 @@
+/*++
+
+Copyright (c) Microsoft Corporation. All rights reserved.
+
+Licensed under the MIT License.
+
+Module Name:
+
+    bestla_gemm.h
+
+Abstract:
+
+    Currently only support Q4 gemm.
+--*/
+
+#pragma once
+
+#include <stdint.h>
+#include <cstddef>
+
+/**
+ * @brief Define compute types of block quantization
+ */
+typedef enum {
```

Comment:
You do not need to use typedef here since this header file would not be used by any C code. In C++ the typedef is unnecessary.

Reply:
removed in dc31102941c12602afaf913b3bd9a095958cb92f